### DISCUSSION TITLE
New Github fork for GrandOrgue

### DISCUSSION THREAD

**Discussion by vpoguru** [GrandOrgue/grandorgue#695] [General]
Is there a plan to merge these two branches?

https://github.com/oleg68/GrandOrgue
https://github.com/GrandOrgue/grandorgue

What is the advantage of one or the other?

**Comment by oleg68** [comment]
> Is there a plan to merge these two branches?

Yes, it is. Now the work is in progress. The source code is already the same in both repos.

> What is the advantage of one or the other?

The second repo is the new *Official* repository of GrandOrgue. But it is still under construction. So now this repo is the only place where you can discuss GrandOrgue.

After https://github.com/GrandOrgue/grandorgue becomes ready, I will switch my activity there. This repo will be archived.
